Lib/gftools/builder/fontc.py

```diff
@@ -65,9 +66,8 @@ def modify_config(self, config: dict):
             config["cleanUp"] = True
             # disable running ttfautohint, because we had a segfault
             config["autohintTTF"] = False
-            # set --no-production-names, because it's easier to debug
             extra_args = config.get("extraFontmakeArgs") or ""
```

Comment:
Isn't this already a way to get extra args to the compiler?

Reply:
well, that's the way if you start with a config. here ttx_diff wants to override the user config and to pass down some extra command line options

Reply:
this is where gftools builder gets called from ttx_diff.py

https://github.com/googlefonts/fontc/blob/8ffba4caa50c68b4fc9916d5e77461e98b9e6f83/resources/scripts/ttx_diff.py#L275-L299

we want to inject some extra flag for fontc/fontmake. Are you suggesting that we modify the input yaml before passing it to gftools?

Reply:
Well, that's how I would do it - it would avoid special-case stuff like this. But either way works. So long as you had considered it and thought this was easier...